yt_dlp/extractor/taptap.py

```diff
@@ -0,0 +1,148 @@
+from .common import InfoExtractor
+from ..utils import (
+    clean_html,
+    int_or_none,
+    parse_qs,
+    str_or_none,
+    traverse_obj,
+    url_or_none,
+)
+
+
+class TapTapIE(InfoExtractor):
+    _VALID_URL = r'https?://www\.taptap\.cn/(?P<section>moment|app)/(?P<id>\d+)'
```

Comment:
Could we also support taptap.io? The site structure is very similar. I think this should be doable with only minimal adjustments.